hazelcast-jet-core/src/main/java/com/hazelcast/jet/Processor.java

```diff
@@ -144,5 +145,29 @@ default boolean isCooperative() {
          */
         @Nonnull
         String vertexName();
+
+        /**
+         * Returns the future to check for cancellation status.
+         * <p>
+         * This is only necessary, if the {@link #complete()} method never returns and blocks
+         * indefinitely, which is only legal for {@link #isCooperative() non-cooperative} processors.
```

Comment:
Could you please format this text so the length of the text line is up to 72 chars? This doesn't include any chars preceding the first char of actual text (it's not about the right margin, but actual text line length).